### PR TITLE
Update troubleshooting section of Readme.md

### PR DESCRIPTION
**Pull request by pbenefice**
**Description of the change**

Just an update of the Readme.

**Benefits**

I'm thinking this might save some time to the following one having this issue. At least I would have loved to see it there for myself.

**Possible drawbacks**

None

**Applicable issues**

None that are still open, but I encountered a few related to this solution 

**Additional information**

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -339,6 +339,10 @@ This would be an example of SMTP configuration using a Gmail account:
 # Troubleshooting
 
 * If you are automatically logged out from the administration panel, you can try deploying SuiteCRM with the environment variable `SUITECRM_VALIDATE_USER_IP=no`
+* If you get a timeout error during the initial setup similar to :
```
Comment (andresbono) [inline README.md]: Can you remove the space before the `:`?

File: README.md [hunk 2]
```diff
@@ -339,6 +339,10 @@ This would be an example of SMTP configuration using a Gmail account:
...
+  > Error executing 'postInstallation': Timeout: did not get to load all resources on this page
+  
+  you can try to increase the value of the `SUITECRM_HTTP_TIMEOUT` environment variable mentionned above.
```
Comment (andresbono) [inline README.md]: Can you capitalize `you`?

`you` -> `You`.

Comment (andresbono) [inline README.md]: `mentionned` -> `mentioned`.

**Comment by andresbono** [comment]
Hi @pbenefice, thanks so much for your contribution! Sorry that we have missed it until now.

**Comment by pbenefice** [comment]
Hi @andresbono, thanks for your review.
I added the requested changes (push forced, sry), and resolved the conversations.
It should by okay.

No problem for the delay, it's not like if it was critical 😄

**Comment by andresbono** [comment]
Merged! Thanks for your contribution!

**Pull request merged**